### PR TITLE
Move FWCore and Configuration python scripts to python3

### PR DESCRIPTION
**Pull request by davidlange6**

### CODE REVIEW

**Comment by Dr15Jones** [comment]
@davidlange6 did you try running the `mk*` scripts to be sure they work under python3?

**Comment by davidlange6** [comment]
I ran at least mkesprod and mkedprod.  any of the others follow a different pattern that I should test out?

**Comment by davidlange6** [comment]
please test

**Comment by Dr15Jones** [comment]
@davidlange6 mkskel is the only outlier I can think of.

**Comment by davidlange6** [comment]
that runs ok as well

**Comment by makortel** [comment]
+1

Reco comparison differences look like the order Paths has changed between the baseline and py3 version of these files, presumably order of dictionary keys in ConfigBuilder or something similar. Py3 build has anyway been validated by PdmV.

File: FWCore/Skeletons/python/utils.py [hunk 1]
```diff
@@ -101,6 +101,7 @@ def capture():
         print("\n### generated code\n")
         print(func)
     # compile python code as exec statement
+    print(func)
```
Comment (srimanob) [inline FWCore/Skeletons/python/utils.py:104]: Why do we need this line also outside debug? If need, should we move it out from debug (line 102, above)?

Comment (davidlange6) [reply FWCore/Skeletons/python/utils.py:104]: good question. I'm not sure how I added that, but it was not intended. Fixing that.

**Comment by davidlange6** [comment]
please test

**Comment by srimanob** [comment]
Please test

**Comment by davidlange6** [comment]
this failed test hit a timeout

**Comment by makortel** [comment]
+1

**Comment by srimanob** [comment]
+Upgrade

**Comment by chayanit** [comment]
+1

**Comment by qliphy** [comment]
+operations

**Comment by qliphy** [comment]
+1

**Comment by qliphy** [comment]
merge

**Pull request merged**